docs/mctpd.md

```diff
@@ -104,3 +104,17 @@ busctl call au.com.codeconstruct.MCTP1 \
 
 Removes the MCTP endpoint from `mctpd`, and deletes routes and neighbour entries.
 
+## D-Bus /au/com/codeconstruct/mctp1/interfaces/<name>
+
+`mctpd` provides a D-Bus path of `/au/com/codeconstruct/mctp1/interfaces`.
+For each known MCTP interfaces, `mctpd` will populate an D-Bus object
+`/au/com/codeconstruct/mctp1/interfaces/<name>`. The D-Bus objects have
+interface `au.com.CodeConstruct.MCTP.Link1`.
```

Comment:
So we have a small inconsistency between `interfaces` and `Link` there. Can we unify?

Reply:
We concerned about `/au/com/codeconstruct/mctp1/interfaces/<name>` and `/au/com/codeconstruct/mctp1/links/<name>` before. We can use `/au/com/codeconstruct/mctp1/links/<name>` for unifying.

Reply:
Yeah, we had discussed this previously, and settled on `/interfaces` (as you have implemented here). However, the dbus interface name still uses `Link`.

Reply:
So what do you want? Change the object path to `/au/com/codeconstruct/mctp1/links/<name>`?

Reply:
no, we should have the interface name be `au.com.codeconstruct.MCTP.Interface1`. The object paths are good as they are.

Reply:
(I'm happy to do that if you prefer, but want to make sure I'm not stepping on your toes by doing so!)

Reply:
Thanks. Please help me do that. So we can reuse this PR.

Reply:
OK, I have added: https://github.com/CodeConstruct/mctp/pull/44/commits/fabb82973cd25bc61d016669f07e94f76df734b4

If that looks good to you, I'll squash that into your change.

Reply:
> OK, I have added: [fabb829](https://github.com/CodeConstruct/mctp/commit/fabb82973cd25bc61d016669f07e94f76df734b4)

The commit LGTM.

> 
> If that looks good to you, I'll squash that into your change.